Server/src/main/java/net/simon987/server/assembly/CPU.java

```diff
@@ -466,6 +468,14 @@ public CpuHardware getHardware(int address) {
         return attachedHardware.get(address);
 
     }
-
-
+    /**
+     * Sets the IP to 0x0200 + Offset and pushes flags then the old IP
+     * 
+     */
+    public void Interrupt(int hw, int offset, boolean retry) {
```

Comment:
Should be interrupt with a lowercase i
What is `retry`?

Reply:
Retry was something I left in for future proofing. Without VM it doesn't really make much sense.

It was so after an instruction failed and interrupted you could fix the problem and restart the instruction. Might just make that the OS's problem.